storageconnector/pgconnector.go

```diff
@@ -5,48 +5,119 @@ import (
 	"fmt"
 	_ "github.com/lib/pq"
 	"regexp"
+	"strings"
 )
 
 type postgreStorage struct {
 	db *sql.DB
 }
 
-func ConnectToPg (username string, dbname string) {
+func ConnectToPg(username string, dbname string) {
 	fmt.Println("Opening database connection.")
-	command := fmt.Sprintf("user=%s dbname=%s sslmode=disable",username,dbname)
+	command := fmt.Sprintf("user=%s dbname=%s sslmode=disable", username, dbname)
 	db, err := sql.Open("postgres", command)
 
 	checkErr(err)
 	fmt.Println("Successfully opened database connection.")
-	db.Exec("CREATE TABLE IF NOT EXISTS features (index int, name text, description text, author text, lastupdate timestamptz, dependencies text);")
+	db.Exec("CREATE TABLE IF NOT EXISTS features (index int, name text, description text, author text, lastupdate timestamptz, dependencies text, snippet text);")
 }
 
-func (store* postgreStorage) disconnectFromPg() {
+func (store *postgreStorage) disconnectFromPg() {
 	store.db.Close()
 }
 
-func (store* postgreStorage) SearchMeta(name *regexp.Regexp) ([]FeatureMeta, error) {
-// TODO: implement (github issue #91)
-	return nil, nil
+func (store *postgreStorage) SearchMeta(name *regexp.Regexp) ([]FeatureMeta, error) {
+	var sql_query string
+	var dep_text string
+	var snippet string
+	sql_query = fmt.Sprintf("select * from features where name ~ %s", name)
+	rows, err := store.db.Query(sql_query)
+	checkErr(err)
+	defer rows.Close()
+	var fms []FeatureMeta
+	for rows.Next() {
+		var f FeatureMeta
+		var index int
+		err := rows.Scan(index, f.Name, f.Description, f.Author, f.UpdatedAt, dep_text, snippet)
+		checkErr(err)
+		fms = append(fms, f)
+	}
+	return fms, nil
+
 }
 
-func (store* postgreStorage) GetMeta(name string) (FeatureMeta, error) {
-// TODO: implement (github issue #91)
+func (store *postgreStorage) GetMeta(name string) (FeatureMeta, error) {
 	var f FeatureMeta
+	var sql_query string
+	var index int
+	var dep_text string
+	var snippet string
+	sql_query = fmt.Sprintf("select * from features where name == %s", name)
+	err := store.db.QueryRow(sql_query).Scan(index, f.Name, f.Description, f.Author, f.UpdatedAt, dep_text, snippet)
+
+	checkErr(err)
+
+	s := strings.Split(dep_text, " ")
+	f.Dependencies = make([]string, len(s))
+
+	for index, value := range s {
+		f.Dependencies[index] = value
+	}
```

Comment:
Why not just `f.Dependencies = s`?

I will most probably rename `s` since it's not clear what is `s` :-)